src/openforms/authentication/contrib/digid_oidc/urls.py

```diff
@@ -0,0 +1,21 @@
+from django.conf.urls import url
+
+from mozilla_django_oidc.urls import urlpatterns
+
+from .views import OIDCAuthenticationCallbackView, OIDCAuthenticationRequestView
+
+app_name = "digid_oidc"
+
+
+urlpatterns = [
+    url(
+        r"^callback/$",
```

Comment:
please use the more modern `django.urls.path`, `django.conf.urls.url` is deprecated and removed in 3.x if I remember correctly